### PR TITLE
chatRemove_aayushi

### PR DESCRIPTION
**Pull request by aayushitiwari03**
Changes to Remove Chat modules unused classes and code.

### CODE REVIEW

File: app/src/main/java/org/merakilearn/MerakiApp.kt [hunk 1]
```diff
@@ -55,8 +56,8 @@ class MerakiApp : Application() {
         val config: Config by inject()
         config.initialise()
 
-        ChatInitializer.initialise(this)
-        subscribeToDefaultTopic()
+//        ChatInitializer.initialise(this)
+//        subscribeToDefaultTopic()
```
Comment (surajsahani) [inline app/src/main/java/org/merakilearn/MerakiApp.kt]: Do not remove subscribeToDefaultTopic()

File: chat/src/main/java/org/navgurukul/chat/core/repo/AuthenticationRepository.kt [hunk 2]
```diff
@@ -27,7 +27,7 @@ class AuthenticationRepository(
     private val appContext: Context
 ) {
 
-    suspend fun getLoginFlow(): LoginFlowResult? = suspendCoroutine {
+    private suspend fun getLoginFlow(): LoginFlowResult? = suspendCoroutine {
```
Comment (surajsahani) [inline chat/src/main/java/org/navgurukul/chat/core/repo/AuthenticationRepository.kt]: Do not remove fake login part

**Review by surajsahani — CHANGES_REQUESTED** [review]
Please remove commented changes

**Comment by surajsahani** [comment]
@aayushitiwari03  Remove entire chat module from compilation process. remove implementation part from build.gradle

File: chat/src/main/java/org/navgurukul/chat/core/repo/AuthenticationRepository.kt (hunk 2 above)
Comment (aayushitiwari03) [reply chat/src/main/java/org/navgurukul/chat/core/repo/AuthenticationRepository.kt]: okay

Comment (aayushitiwari03) [reply chat/src/main/java/org/navgurukul/chat/core/repo/AuthenticationRepository.kt]: I didn't remove it I just made this function private please recheck.

**Comment by aayushitiwari03** [comment]
Chat is already removed and merged into development along with updated libraries

**Pull request closed without merging**